src/scldm/download_artifacts.py

```diff
@@ -381,7 +381,7 @@ def main():
     )
 
     try:
-        errors = download_artifacts(destination=args.destination, credentials=credentials, groups=groups)
+        _ = download_artifacts(destination=args.destination, credentials=credentials, groups=groups)
```

Comment:
oops, yeah, thanks, or you can just do `download_artifacts(...)` since I'm not using it

Reply:
nvm, I activate pre-commit ci from now on